pkg/db/models/releases.go

```diff
@@ -112,7 +112,7 @@ type ReleaseJobRun struct {
 
 	ReleaseTag     ReleaseTag `json:"release_tag" gorm:"foreignKey:release_tag_id"`
 	ReleaseTagID   string     `gorm:"column:release_tag_id"`
-	Name           uint       `json:"name" gorm:"column:prow_job_run_id"` // TODO: this could use a rename to ProwJobRunID
+	Name           uint       `json:"name" gorm:"column:prow_job_run_id,index:prow_job_run_id_idx"` // TODO: this could use a rename to ProwJobRunID
```

Comment:
A plain index is just a tool to help lookups, this won't guarantee uniqueness.  You want a unique index: https://gorm.io/docs/indexes.html

Reply:
change pushed to use `index:unique`.